postgres/datadog_checks/postgres/config.py

```diff
@@ -58,6 +59,13 @@ def __init__(self, instance):
         self.custom_metrics = self._get_custom_metrics(instance.get('custom_metrics', []))
         self.max_relations = int(instance.get('max_relations', 300))
 
+        # Deep Database monitoring adds additional telemetry for statement metrics
+        self.deep_database_monitoring = is_affirmative(instance.get('deep_database_monitoring', False))
+        # Support a custom view when datadog user has insufficient privilege to see queries
+        self.pg_stat_statements_view = instance.get('pg_stat_statements_view', 'pg_stat_statements')
+        self.escape_query_commas_hack = is_affirmative(instance.get('escape_query_commas_hack', True))
+        self.statement_metric_limits = instance.get('statement_metric_limits', None)
```

Comment:
New config options need to be reflected in the config specs. If this is an experimental feature you can say so in the config description